website/docs/guides/release-compatibility.html.markdown

```diff
@@ -50,6 +50,7 @@ Quick at-a-glance access to Aviatrix Terraform provider's release compatibility
 | 0.12              | 2.7                      | UserConnect-5.1              |
 | 0.12              | 2.8                      | UserConnect-5.1, 5.2         |
 | 0.12              | 2.9                      | UserConnect-5.2              |
+| 0.12              | 2.9.1                    | UserConnect-5.2.2122         |
```

Comment:
We should probably cherry-pick this onto the stable website branch

Reply:
I will do it.